chia/solver/solver_rpc_api.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar, Optional, cast
+
+from chia.rpc.rpc_server import Endpoint, EndpointResult
+from chia.solver.solver import Solver
+from chia.util.ws_message import WsRpcMessage
+
+
+class SolverRpcApi:
+    if TYPE_CHECKING:
+        from chia.rpc.rpc_server import RpcApiProtocol
+
+        _protocol_check: ClassVar[RpcApiProtocol] = cast("SolverRpcApi", None)
+
+    def __init__(self, solver: Solver):
+        self.service = solver
+        self.service_name = "chia_solver"
+
+    def get_routes(self) -> dict[str, Endpoint]:
+        return {
+            "/solve": self.solve,
```

Comment:
you removed support for this RPC in `SolverRpcClient`, did you leave it in here on purpose?